crates/language/src/language_settings.rs

```diff
@@ -292,6 +293,8 @@ pub enum FormatOnSave {
         /// The arguments to pass to the program.
         arguments: Arc<[String]>,
     },
+    // Files should be formatted using code actions executed by language servers.
```

Comment:
This should be a doc comment:

```suggestion
    /// Files should be formatted using code actions executed by language servers.
```